Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/core/PersonCertificatesProvider.kt

```diff
@@ -113,29 +76,25 @@ class PersonCertificatesProvider @Inject constructor(
         personCertificatesSettings.setCurrentCwaUser(personIdentifier)
     }
 
-    val personsBadgeCount: Flow<Int> = personCertificates
-        .map { persons -> persons.sumOf { it.badgeCount } }
+    val personsBadgeCount: Flow<Int> = personCertificates.map { persons -> persons.sumOf { it.badgeCount } }
 
     private fun Set<VaccinatedPerson>.boosterBadgeCount(
```

Comment:
Maybe rename this method to `hasBoosterBadge` since we are returning `Boolean` now.